doc/userguide/rules/ipaddr.rst

```diff
@@ -0,0 +1,29 @@
+IP Addresses Match
+==================
+
+Matching on IP addresses can be done via the IP tuple parameters or via the iprep keywords (see :doc:`/rules/ip-reputation-rules`).
+Some keywords providing interaction with dataset are also available.
+
+ip.src
+------
+
+The `ip.src` keyword is a sticky buffer to match on source IP address. It is doing a match on the binary representation
```

Comment:
Suggestion: instead of  "It is doing a match", "It matches on..."

Reply:
Come on, I'm paid by the character :P

Reply:
oh, wait, then I need you to expand the explanation in one more paragraph :D